contracts/registries/IPAssetRegistry.sol

```diff
@@ -33,8 +33,8 @@ contract IPAssetRegistry is IIPAssetRegistry, IPAccountRegistry, Governable {
     constructor(
         address erc6551Registry,
         address ipAccountImpl,
-        address governance
-    ) IPAccountRegistry(erc6551Registry, ipAccountImpl) Governable(governance) {}
+        address accessManager
+    ) IPAccountRegistry(erc6551Registry, ipAccountImpl) AccessManaged(accessManager) {}
```

Comment:
Do we need AccessManaged (or Governable) for IPAssetRegistry? IPAssetRegistry is not upgradeable

Reply:
AccessManaged (or formerly Governable) has to do with admin method gating through roles, not upgradeability. You are correct indeed on not needing it here, since there are no restricted method, must have been a holdout from before. Good catch!

Reply:
3c7cc52